src/buza/models.py

```diff
@@ -0,0 +1,69 @@
+from __future__ import unicode_literals
```

Comment:
Note that we can drop `unicode_literals` and other Python 2 compatibility code, since we're no longer aiming for Python 2 compatibility.